openjdk/src/test/java/org/conscrypt/NativeCryptoTest.java

```diff
@@ -2914,9 +2919,22 @@ public void test_RSA_public_decrypt_NullArgumentFailure() throws Exception {
      * openssl rand -hex 16
      */
     private static final byte[] AES_128_KEY = new byte[] {
```

Comment:
We usually have a `// clang-format off ` for this kind of variables. Just fyi for future changes.